lib/versionista.js

```diff
@@ -610,9 +610,25 @@ function getPagingUrls (window) {
 function getPageDetailData (window) {
   const xpathRows = xpath(window.document, "//div[contains(text(), 'URL')]/../../../following-sibling::tbody/tr");
   return xpathRows.map(row => {
-    const updateTimeText = parseFloat(xpathNode(row, "./td[9]").textContent.trim());
+    let lastChange = null;
+    const updateTimeText = xpathNode(row, "./td[8]").textContent.trim();
+    if (!/^(\d*(\.\d*)?|[\-–—])$/.test(updateTimeText)) {
+      throw new Error(oneLine(`Expected update time to be a numeric timestamp,
+        but got "${updateTimeText}" (${window.location.href})`));
+    }
+    else {
+      const timestamp = parseFloat(updateTimeText);
+      if (!Number.isNaN(timestamp)) {
+        lastChange = new Date(1000 * updateTimeText);
```

Comment:
Not sure if there's any advantage, but should we use `timestamp` here instead of `updateTimeText` because it's already a float? 

Otherwise, LGTM! 🔢 👍 📅

Reply:
That is a typo.

Reply:
Ha, thought you were playing it wild with type coercion 😂